lib/active_merchant/billing/gateways/payway_dot_com.rb

```diff
@@ -56,7 +56,10 @@ class PaywayDotComGateway < Gateway
       SCRUB_PATTERNS = [
         %r(("password\\?":\\?")[^\\]+),
         %r(("fsv\\?":\\?")\d+),
-        %r(("accountNumber\\?":\\?")\d+)
+        %r(("fsv\\?": \\?")\d+),
+        %r(("accountNumber\\?":\\?")\d+),
+        %r(("accountNumber\\?": \\?")[^\\]+),
+        %r(("Invalid account number: )\d+)
```

Comment:
I'm not fluent in Regex, but are you sure we don't need operators similar to the others above for this?